src/components/styles/Button.styled.ts

```diff
@@ -0,0 +1,52 @@
+import styled from 'styled-components'
+
+export const BaseButton = styled.div<{ fontSize?: string; isActive?: boolean }>`
+  width: fit-content;
+  justify-content: center;
+  align-items: center;
+  .ant-btn-text {
+    color: ${(props) => props.theme.palette.common.black};
+    font-size: ${(props) => props.fontSize};
+    border: 0;
+    border-bottom: transparent 2px solid;
+    background-color: transparent;
+    transition: background-image 0.4s ease-in-out;
+    &:hover {
+      background-position: 50% 1.2em;
+      color: ${(props) => props.theme.palette.primary};
+      //css solution for styled underline
+      background-image: linear-gradient(
+        to bottom,
+        ${(props) => props.theme.palette.primary},
+        ${(props) => props.theme.palette.primary}
+      );
+      background-repeat: no-repeat;
+      // background-position: 50% 2em;
```

Comment:
should this comment be removed?

Reply:
e4dd61daa96632369ea971d797017b2fb05ebfb3